packages/env/lib/download-sources.js

```diff
@@ -128,7 +128,9 @@ async function downloadGitSource( source, { onProgress, spinner, debug } ) {
 	}
 
 	log( 'Fetching the specified ref.' );
-	await git.fetch( 'origin', source.ref );
+	await git.fetch( 'origin', source.ref, {
+		'--tags': null
+	} );	
```

Comment:
the linter is complaining about some whitespace after this line

Reply:
Okay just ran it through a linter. Hope the the tests passes this time around.